operator/v1/types_machineconfiguration.go

```diff
@@ -32,6 +32,18 @@ type MachineConfigurationSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
 
 	// TODO(jkyros): This is where we put our knobs and dials
+
+	// bootImageUpdateMode specifies whether or not the MCO
+	// will attempt to update boot images for every machineset
+	// If unset, this property defaults to disabled
+	// +optional
+	BootImageUpdateMode BootImageUpdateModeEnum `json:"bootImageUpdateMode,omitempty"`
+
+	// bootImageUpdateEnrolledMachineSets contains a list of machinesets that will be enrolled for updates
+	// +listType=atomic
+	// +nullable
+	// +optional
+	BootImageUpdateEnrolledMachineSets []string `json:"bootImageUpdateEnrolledMachineSets"`
```

Comment:
Why is this a list of names, why not use something like a selector like you would for a deployment to select pods, that would be move flexible/able to evolve over time